src/Util/util.h

```diff
@@ -264,7 +264,7 @@ bool end_with(const std::string &str, const std::string &substr);
 //拼接格式字符串
 template<typename... Args>
 std::string str_format(const std::string &format, Args... args) {
-#if __cplusplus > 202002L
+#if __cpp_lib_format
```

Comment:
According to [this](https://en.cppreference.com/w/cpp/feature_test#Library_features), the macro was defined in `<format>`, and `[Text formatting](https://en.cppreference.com/w/cpp/compiler_support/20)` was introduced to gcc since 13.

Should we check and include the `format` to make this work?

Reply:
That might be an issue for some compilers. Although on my system, format, and the macro was already accessible without  including <format>. this can also be that the header is included elsewhere in the library

Reply:
@alexliyu7352 PTAL